src/model/SetpointManager.cpp

```diff
@@ -177,6 +179,22 @@ namespace model {
       return boost::none;
     }
 
+    ComponentType SetpointManager_Impl::componentType() const {
+      return ComponentType::None;
+    }
+
+    std::vector<FuelType> SetpointManager_Impl::coolingFuelTypes() const {
+      return {FuelType::Electricity};
```

Comment:
why does a setpoint manager have a cooling fuel type?

Reply:
That would be a problem between the keyboard and the chair. I probably copy pasted, shame on me!